### ISSUE TITLE
Specify Breakglass users and/or group for correct reporting

### ISSUE THREAD

**Issue by SecDudz** [Jhope188/ca-policy-analyzer#44]
Suggestion: Add logic where if a group containing the name "Breakglass" / "Break-glass" / "Break glass" is used as an exclusion in a policy, then the Breakglass exclusion-recommendation is fullfilled.

Then the scoring and recommendations in CA Policy Analyzer will be more correct.
(I use an Entra **group that is role-assignable**  containing the breakglass accounts that is excluded in all policies. It means only GA or Privileged Role Admin can manage members)

Example from the Analysis report (see Excluded Groups marked with green):
<img width="856" height="322" alt="Image" src="https://github.com/user-attachments/assets/ad1477b4-3650-4aa3-8814-2bd9b9339e15" />

**Comment by dermo-blast** [comment]
+1 on the problem — I've hit the same thing, and I think the cause is a bit deeper than the name matching, so name patterns alone may not fix it.

## Why a correctly-excluded group still doesn't get credit

`identifyBreakGlass()` in `src/lib/analyzer.ts` counts how often each user/group appears in an exclusion and then keeps **only the single highest**:

```js
if (!p.conditions.users.includeUsers.includes("All")) continue;   // ~line 2246
...
if (!primary || data.count > primary.count) primary = { id, ...data };  // ~line 2270
```

Two consequences:

1. **Winner-takes-all.** If any other group is excluded from more policies than the break-glass group — a service-account group, a persona/exclusion group, a sync-account group — that group becomes "the break-glass candidate" and the real one is never considered. @SecDudz, if your group is excluded everywhere and still isn't credited, this is the most likely reason: something else simply out-counts it.
2. **Only All-Users policies are counted.** Exclusions on policies that don't target All Users never contribute, so a group excluded from a mix of scoped policies can score lower than it should.

Neither is affected by naming.

## Name matching helps, but can't be the mechanism

I tried the proposed patterns (`breakglass` / `break-glass` / `break glass`) against realistic group names:

```
MISS   CA-Persona-EmergencyAccess
MISS   Emergency Access Accounts
MATCH  BreakGlass-Accounts
MATCH  SG-Break Glass
```

Tenants built on the common persona baselines (Claus Jespersen's framework, Kenneth van Surksum's and Joey Verlinden's variants) frequently name this group something like `CA-Persona-EmergencyAccess` — no "breakglass" token anywhere. Those tenants would still get the wrong answer.

Widening the patterns to catch `emergency` then risks the opposite error: a group called "Emergency Response Team" isn't a break-glass group, and quietly marking the recommendation satisfied is worse than leaving it unsatisfied, because it's a silent false pass on an availability control.

## Also: there should be room for two

Microsoft recommends [two emergency access accounts](https://learn.microsoft.com/entra/identity/role-based-access-control/security-emergency-access). Because the detection keeps a single `primary`, the second one can't be reported today no matter how it's named — worth folding into whatever fix lands here.

## Suggestion

The issue title already points at the strongest fix — **"Specify Breakglass users and/or group"**. Letting the admin declare it outright sidesteps all of the above: no inference, no pattern list to maintain, no false pass. Something like:

1. **Explicit selection** (UI field, and a matching field in the offline export) — declared truth wins over inference.
2. **Naming heuristics as a hint, not a verdict** — use them to *pre-select* a likely candidate the admin confirms, rather than to silently satisfy the recommendation. That keeps @SecDudz's idea, which is a good one, without the false-pass risk.
3. **Support more than one** account/group, and report coverage per account.

Happy to help with any of it if useful — and happy to defer entirely if you'd rather keep the scope to the name matching for now. @SecDudz, does an explicit specification cover your scenario, or is automatic detection important for you (e.g. running this across many tenants where you'd rather not configure each one)?
